ksp_plugin/plugin.hpp

```diff
@@ -185,6 +186,21 @@ class Plugin {
           Transforms<Barycentric, Rendering, Barycentric>*> const transforms,
       Position<World> const& sun_world_position) const;
 
+  // Not const because of the stupid global variable
+  // |transforms_are_operating_on_predictions_|.
+  virtual RenderedTrajectory<World> RenderedPrediction(
```

Comment:
I don't think there's a need to comment the accessors that have a trivial behaviour.
Done otherwise.